### PR TITLE
fn 0.6.4

### PR DESCRIPTION
**Pull request by chenrui333**
[`action-homebrew-bump-formula`](https://github.com/dawidd6/action-homebrew-bump-formula)

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by nandahkrishna** [comment]
The test seemed to be flaky – when I re-ran CI, it failed for other versions of macOS (not just 11) with similar errors. The delay/sleep seems to make it pass, not sure if that's ideal though.

File: Formula/fn.rb [hunk 1]
```diff
@@ -45,6 +45,7 @@ def install
       ENV["FN_REGISTRY"] = "fnproject"
       expected = "Successfully created app:  myapp"
       output = shell_output("#{bin}/fn create app myapp")
+      sleep 1
```
Comment (SMillerDev) [inline Formula/fn.rb:48]: Why would it need to sleep before validating a string?

Comment (nandahkrishna) [reply Formula/fn.rb]: That was in the wrong place, at the very least 🤦🏻‍♂️.

Comment (carlocab) [reply Formula/fn.rb]: > Why would it need to sleep before validating a string?

To be fresh and energised for the tedious work of string validation

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**